addons/mail/static/src/components/chatter/chatter_tests.js

```diff
@@ -452,6 +461,116 @@ QUnit.test('do not post message with "Enter" keyboard shortcut', async function
     );
 });
 
+QUnit.test("Display suggested recipients list", async function (assert) {
+    assert.expect(10);
+
+    this.data['res.partner'].records.push({ id: 100 });
+    await this.start({
+        async mockRPC(route, args) {
+            if (route === '/mail/get_suggested_recipients') {
+                return Promise.resolve(
+                    {
+                        100: [
+                            [
+                                false,
+                                "john@example.com",
+                                'CC email',
+                            ],
+                            [
+                                false,
+                                "fred@example.com",
+                                'CC email',
+                            ],
+                            [
+                                100,
+                                "second partner",
+                                "CC email"
+                            ],
+                            [
+                                false,
+                                "carl@example.com",
+                                'CC email',
+                            ],
+                        ]
+                    });
+            }
+            return this._super(...arguments);
+        },
+    });
+    const chatter = this.env.models['mail.chatter'].create({
+        threadId: 100,
+        threadModel: 'res.partner',
+    });
+    await this.createChatterComponent({ chatter });
+
+    await afterNextRender(() =>
+        document.querySelector(`.o_ChatterTopbar_buttonSendMessage`).click()
+    );
+
+    assert.containsOnce(
+        document.body,
+        '.o_ComposerSuggestedRecipientList',
+        "Must containt a suggested recipient list"
+    );
+    assert.containsN(
+        document.body,
+        '.o_ComposerSuggestedRecipient',
+        3,
+        "at start, the list should not contain more than 3 suggested recipients"
+    );
+    const title = document.querySelector('.o_ComposerSuggestedRecipientList .o_ComposerSuggestedRecipient:first-child').getAttribute('title');
+    assert.strictEqual(
+        title,
+        "Add as recipient and follower (reason: CC email)",
+        "reason must be present in the suggested recipient title",
+    );
+    const checkboxUnchecked = document.querySelector('.o_ComposerSuggestedRecipientList .o_ComposerSuggestedRecipient:first-child input[type=checkbox]');
+    assert.notOk(
+        checkboxUnchecked.checked,
+        "suggested recipient without a partner should not be checked by default",
+    );
+    const checkboxChecked = document.querySelector('.o_ComposerSuggestedRecipientList .o_ComposerSuggestedRecipient input.o-has-partner');
```

Comment:
Same remark on selector as previous remarks on selectors.
In general having whitespaces/tags/CSS pseudos in selector is bad, because too dependent on layout and/or style.